Python/bltinmodule.c

```diff
@@ -206,6 +298,9 @@ builtin___build_class__(PyObject *self, PyObject **args, Py_ssize_t nargs,
     Py_DECREF(meta);
     Py_XDECREF(mkw);
     Py_DECREF(bases);
+    if (modified_bases) {
+        Py_DECREF(old_bases);
```

Comment:
Just
```c
Py_XDECREF(old_bases);
```

Reply:
This is not really possible, since this code is also executed on success, and in this case if there were no conversion, then ``bases`` should be ``DECREF``ed exactly once.